### PR TITLE
Multichain

### PR DESCRIPTION
**Pull request by johnwhitton**

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        'error',
+        { ignores: ['modules'] }
+      ],
+      'node/no-unpublished-import': 0,
+      'node/no-missing-import': ['error', {
+        tryExtensions: ['.js', '.json', '.node', '.ts']
+      }]
+    }
+  }
+  Footer
```
Comment (polymorpher) [inline .eslintrc.js:28]: ?

File: .env.example [hunk 2]
```diff
@@ -0,0 +1,8 @@
+LOCAL_URL="http://localhost:8545"
+LOCAL_MNEMONIC="test test test test test test test test test test test junk"
+TESTNET_URL="https://api.s0.b.hmny.io"
+TEST_MNEMONIC=""
+MAINNET_URL="https://api.s0.t.hmny.io/"
+S1_URL="https://api.s1.t.hmny.io/"
+MNEMONIC=""
+TLD = "country"
```
Comment (polymorpher) [inline .env.example:8]: bad formatting

File: contract/.env.sample [hunk 3]
```diff
@@ -3,5 +3,6 @@ LOCAL_MNEMONIC="test test test test test test test test test test test junk"
 TESTNET_URL="https://api.s0.b.hmny.io"
 TEST_MNEMONIC=""
 MAINNET_URL="https://api.s0.t.hmny.io/"
+S1_URL="https://api.s1.t.hmny.io/"
 MNEMONIC=""
-
+TLD = "country"
```
Comment (polymorpher) [inline contract/.env.sample:8]: same here

File: contract/deploy/ensDeployer.ts [hunk 4]
```diff
@@ -61,14 +62,13 @@ const f = async function (hre: HardhatRuntimeEnvironment) {
...
     ReverseRegistrar: await ensDeployer.reverseRegistrar(),
-    BaseRegistrar: await ensDeployer.baseRegistrar(),
     MetadataService: await ensDeployer.metadataService(),
     NameWrapper: await ensDeployer.nameWrapper(),
     ETHRegistrarController: await ensDeployer.registrarController(),
     PublicResolver: await ensDeployer.publicResolver(),
     UniversalResolver: await ensDeployer.universalResolver(),
     Multicall: await Multicall.address,
-  }) + '\'')
+  } , null, 2) + '\'')
```
Comment (polymorpher) [inline contract/deploy/ensDeployer.ts:71]: we actually need this to be in a single line, to be used in ens-app-v3 env

File: .env.example (hunk 2 above)
Comment (polymorpher) [reply .env.example:8]: actually this file should be removed

**Comment by polymorpher** [comment]
Merging to a local branch to revise before merging

File: .eslintrc.js (hunk 1 above)
Comment (johnwhitton) [reply .eslintrc.js:28]: Copy and paste error

File: contract/deploy/ensDeployer.ts (hunk 4 above)
Comment (johnwhitton) [reply contract/deploy/ensDeployer.ts:71]: I tested multiline .env.local in ens-app-v3 and it seems to parse it ok.

File: contract/.env.sample (hunk 3 above)
Comment (johnwhitton) [reply contract/.env.sample:8]: Good catch thanks for fixing

File: .env.example (hunk 2 above)
Comment (johnwhitton) [reply .env.example:8]: Agree only need one file, I think we have used env.example in other repos like ens-app-v3 and .1.country

**Pull request merged**